release-notes/10.0/preview/preview7/dotnetmaui.md

```diff
@@ -2,12 +2,204 @@
+- **Vibration and HapticFeedback**: New `IsSupported` API to check platform support
+- **Windows**: Added API to enable/disable minimize and maximize buttons on Windows
+- **Shell navigation**: Added `Shell.SetNavBarVisibilityAnimationEnabled` property for controlling navigation bar visibility animations
+- **TabbedPageManager**: Made TabbedPageManager public for advanced customization scenarios
+- **StackNavigationManager**: Exposed public APIs for StackNavigationManager on Android
+
+## Deprecated API Removals
+
+As part of .NET 10, several deprecated APIs have been removed:
+
+- **Accelerator class**: Removed from Microsoft.Maui.Controls
+- **ClickGestureRecognizer**: Removed in favor of `TapGestureRecognizer`
+- **Page.IsBusy**: Marked as obsolete
+- **Compatibility.Layout**: removed dependency
+
+## Control and Layout Improvements
+
+This release includes numerous fixes and improvements across controls and layouts.
+
+## .NET for Android
+
+This release includes continued integration with multiple .NET runtimes, and several bug fixes:
+
+- Add `$(EnableDiagnostics)` cross-platform MSBuild property, to align with iOS and WASM.
+- Fixe an issue with `=` symbol in Android environment variables.
+- Fixed an issue where certain Java libraries were not redistributed in NuGet packages.
+- Add support for API 36 in various Visual Studio dropdowns & menus.
+- Changes to the "Java GC Bridge" to support multiple runtimes.
+
+A detailed list can be found on the [dotnet/android GitHub releases](https://github.com/dotnet/android/releases/).
+
+## .NET for iOS, Mac Catalyst, macOS, tvOS
+
+This release includes Xcode 26 Beta 4 support for targeting .NET 9. We will include builds for .NET 10 in a subsequent release. To use these new bindings, target `net9.0-ios26` and/or `net9.0-maccatalyst26` and include `<NoWarn>$(NoWarn);XCODE_26_0_PREVIEW</NoWarn>` in your project file.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net9.0-ios26</TargetFramework>
+    <NoWarn>$(NoWarn);XCODE_26_0_PREVIEW</NoWarn>
+    <!-- rest of your configuration -->
+  </PropertyGroup>
+</Project>
+```
+
+Also in this release are significant improvements to binding generation, runtime performance, and API coverage:
+
+- **New Binding Generator (RGen)**:
+  - Enhanced source generator for better binding performance and maintainability
+  - Added support for nested classes, categories, and constructors
+  - Improved async method generation with proper Task-based patterns
+  - Better memory management and semantic model usage
+  - Support for strong dictionary properties and weak delegate patterns
+
+- **Runtime and Interop Improvements**:
+  - Reworked `NSObject` data storage for better performance
+  - Enhanced P/Invoke handling and native library integration
+  - Improved delegate signature consistency with nullability annotations
+  - Better handling of Action-based delegates across frameworks
```

Comment:
These two have no actual effect yet, they will only take effect once we're using the new generator (rgen).